docs/Help-and-Support.md

```diff
@@ -0,0 +1,120 @@
+
+:::tip
+Please, note, that typically signing up is required to join AlmaLinux discussion spaces.
+:::
+
+### Chats
+
+AlmaLinux Chats allows users to communicate on various channels, each with its specific topics like Packaging, Cloud, and Documentation. If you have a question or need help, post a question in the [*General*](https://chat.almalinux.org/almalinux/channels/town-square) channel or in the relevant channel for assistance.
+
+* [Mattermost Community Chat](https://chat.almalinux.org/)
+* [IRC Real Time Chat](https://web.libera.chat/#almalinux)
+
+### Forums 
+
+AlmaLinux OS also provides forums for sharing information and assistance inside the Community. Subforum based on categories for the convenience of locating a relevant one.
+
+* [AlmaLinux Forums](https://forums.almalinux.org/) 
+
+### Mailing Lists
+
+You can receive information, announcements, news letters, and updates over email by subscribing to the [AlmaLinux Mailing Lists](http://lists.almalinux.org/).
+
+### Social Media
+
+If you want to stay up-to-date, stay tuned on AlmaLinux Social Media:
+
+* [Reddit Community](https://www.reddit.com/r/AlmaLinux/) 
+* [X/Twitter](https://twitter.com/AlmaLinux)
+* [Facebook](https://www.facebook.com/AlmaLinux/)
+* [LinkedIn](https://www.linkedin.com/company/almalinuxos/)
+* [Mastodon](https://fosstodon.org/@almalinux)
+
+## Chat Policies
+
+We kindly ask any Community member to:
+* Be respectful and polite to others.
+* Refrain from using offensive language or harassing others.
+* Be mindful of cultural and social sensitivities when engaging in conversation.
+* Avoid sharing personal information or disclosing confidential details.
+* Respect each other's diverse skills and abilities.
+* For those, who are sharing their expertise, provide as much detail as possible to enhance understanding and collaboration among all members.
+* Report any violations of the rules to the AlmaLinux OS Foundation.
+
+## Getting Help Recommendations
+
+For the convenience of all Community members and a faster response, we advise minding these recommendations when posting a question/issue:
+* Formulate a question/problem. It should be formulated clearly and easily to understand.
+* Provide as many details including software/hardware information as possible.
+* Provide any logs that might be relevant and helpful.
+* [Provide any command output](#providing-an-output) that might be relevant and helpful.
+
+### Providing an output
+
+When asking for assistance, we kindly ask you to use the mentioned below ways of sharing any output.
+
+Note, that it's preferable and recommended to use `bash` syntax.
+
+#### paste.almalinux.org
+
+We suggest you use the [https://paste.almalinux.org/](https://paste.almalinux.org/) service for sharing your outputs. 
+
+:::warning
+Please, note, that outputs shared via [https://paste.almalinux.org/](https://paste.almalinux.org/) expire 1 hour to 1 week.
```

Comment:
We can allow 1 day - indefinite. I just need to make sure no one has existing pastes that they need (as its stored in memory ATM). 

cc @jonathanspw

Reply:
This has been adjusted to allow longer expiry for pastes + persistence. You can safely remove this warning now.

<img width="373" alt="image" src="https://github.com/AlmaLinux/wiki/assets/196810/5a227d85-7931-42e8-b6e2-33d07370959d">